Clustering/KMeans/src/main/java/org/tribuo/clustering/kmeans/KMeansOptions.java

```diff
@@ -41,10 +41,10 @@
     @Option(longName = "kmeans-num-centroids", usage = "Number of centroids in K-Means. Defaults to 10.")
     public int centroids = 10;
     /**
-     * Distance function in K-Means. Defaults to EUCLIDEAN.
+     * Distance function in K-Means. Defaults to L2 (EUCLIDEAN).
      */
-    @Option(longName = "kmeans-distance", usage = "Distance function in K-Means. Defaults to EUCLIDEAN.")
-    public Distance distance = Distance.EUCLIDEAN;
+    @Option(longName = "distance-type-function", usage = "The type of distance function to use for various distance calculations.")
```

Comment:
Can you leave "kmeans" in this argument name? The options classes are designed to be combined (so you could have a single `main` that had both `HdbscanOptions` and `KMeansOptions` in it, so they need names that show which trainer they refer to. I'd missed that the `HdbscanOptions` argument names are not prefixed with `hdbscan` earlier, but they should be.

Reply:
Thanks for this explanation, that make sense.